plugin/storage/factory_config.go

```diff
@@ -56,13 +57,19 @@ func FactoryConfigFromEnvAndCLI(args []string, log io.Writer) FactoryConfig {
 	if spanStorageType == "" {
 		spanStorageType = cassandraStorageType
 	}
-	depStoreType := os.Getenv(DependencyStorageTypeEnvVar)
-	if depStoreType == "" {
-		depStoreType = spanStorageType
+	spanWriterTypes := strings.Split(spanStorageType, ",")
+	if len(spanWriterTypes) > 1 {
+		fmt.Fprintf(log, "WARNING: Since multiple storage types have been listed,"+
+			"the first span storage type listed (%s) will be used for reading and archiving.", spanWriterTypes[0])
+	}
+	depStorageType := os.Getenv(DependencyStorageTypeEnvVar)
```

Comment:
Should this come from viper?

Reply:
All the main.go files initialize viper after setting up this initial factory.

Reply:
If the factories can be created within the Cobra's command, it can also receive a `zap.Logger`, avoiding the usage of `fmt.Fprintf`

Reply:
The reason is to avoid each storage type adding it's flags to cli

Reply:
That affects only the contents of `--help`, right?